scripts/bake-ide-ami.ps1

```diff
@@ -468,9 +468,53 @@ $jsonObject = @"
                 $ChefRecipe = "VLWebServer::IDEBaseAzure"
             }
 
+            # 163204: Generate PFX using secret from Azure Vault
             # Make sure the session is initialised correctly
             ReConnect-Session
 
+            # set the license key password
+            $script:licensekeypassword = $Script:password;
```

Comment:
This code is incorrect:
```
            $script:licensekeypassword = $Script:password
```
The `script init-baking-vars.ps1` sets this up 
```
$script:licensekeypassword = $ENV:cloud_license_key
```
So your environment needs to declare a variable called `cloud_license_key`. 

I think you can set it to any value as its a password for the intermediary PFX file.

Reply:
Hi @robe070 
Yes, I had reverted this change and using the environment variable for setting the license key password.